setup.py

```diff
@@ -59,10 +91,15 @@ def get_basesources(name, osname):
 # Treating some special cases like Octave seems to be the right thing to do,
 # PSYCH_LANGUAGE setting is self-explanatory:
 base_macros = [('PTBOCTAVE3MEX', None), ('PSYCH_LANGUAGE', 'PSYCH_PYTHON')]
-# Disabled: This would enable Py_LIMITED_API, to allow to build one set of modules for all versions of
-# Python >= 3.2. The downside is loss of important functionality [PsychRuntimeEvaluateString()) does not work!].
-# Also, we have build failure on at least Ubuntu 18.04 LTS with Python 3.6, so it is a no-go on Linux for now!
-#base_macros = [('PTBOCTAVE3MEX', None), ('PSYCH_LANGUAGE', 'PSYCH_PYTHON'), ('Py_LIMITED_API', None)]
+
+# if desired by wheel-builder and building on py >= 3.7, build as a limited API wheel.
+if sys.version_info >= (3, 7) and os.environ.get('PTB_LIMITED_WHEEL') == '1':
```

Comment:
I do wonder if it would make more sense to make the limited api an opt-out instead of an opt-in, so it gets used and tested by default, so it can't break as easily if one changes code and doesn't test with limited api opt-in? I doubt there's meaningful overhead due to the limited api. This could be done in a followup commit though if one wants to change it.